lib/remap-lang/grammar.pest

```diff
@@ -83,7 +83,7 @@ regex_flags =  { ("i" | "x" | "m")* }
 
 // Other ----------------------------------------------------------------------
 
-ident  = @{ ASCII_ALPHANUMERIC ~ (ASCII_ALPHANUMERIC | "_")* }
+ident  = @{ !reserved_keyword ~ ASCII_ALPHANUMERIC ~ (ASCII_ALPHANUMERIC | "_")* }
```

Comment:
This is preventing any identifier that starts with a reserved keyword. So the variables `forest` and `iffy` are also disallowed. I think that's overkill and needs to be allowed.

It's a pain. I'm struggling to think of a quick fix to that...

Reply:
Oh, also `ident` is used as path fields, which shouldn't have the reserved keyword restriction. I should be able to do `.foo.self.if.for = 9`.

Reply:
Thanks for catching these 👍